docs/_posts/2017-10-19-diversity-open-source.markdown

```diff
@@ -0,0 +1,44 @@
+---
+title: "Diversity in Open Source, and Jekyll's role in it"
+date: 2017-10-19 21:33:00 +0200
+author: pup
+categories: [community]
+---
+
+Open Source has a problem with diversity. GitHub recently conducted a [survey](http://opensourcesurvey.org/2017) that revealed that 95% of the respondents were male, making the demographic even worse than in tech overall. Every other week, I seem to hear of another case of a maintainer engaging in targeted harassment against minorities. People somehow deem it completely okay to let these things slide, though.
+
+So yeah, there's a bunch of big problems plaguing open source. Fortunately, there's a couple of things we can do to make it easier and more comfortable for people, especially minorities, that have never before contributed to any open source project before, to contribute to our projects.
+
+## Add a Code of Conduct, and enforce it
+
+This might seem like one of the easiest steps to do, but it actually requires a lot of dedication to pull through with. Basically, a Code of Conduct is a document detailing what is and what isn't acceptable behavior in your project. A good Code of Conduct also details enforcement procedures, that means how the person violating the Code of Conduct gets dealt with. This is the point at which I've seen a looooot of projects fail. It's easy enough to copy-paste a Code of Conduct into your project, but it's just, if not more important to be clear on how to enforce it. Inconsistent, or worse, nonexistent enforcement is just going to scare off newcomers even more!
```

Comment:
It looks like there's a dangling "it's just" in the second-to-last sentence there. "It's easy enough .. but it's more important to be clear with how you enforce it."

Reply:
The little copy editor in me likes slightly different punctuation in the last sentence. "Inconsistent—or worse, non-existent—enforcement is just ..." Definitely unnecessary, I just like emdashes. 😹